systemd/systemd.go

```diff
@@ -85,12 +85,17 @@ var JournalctlCmd = jctl
 // Systemd exposes a minimal interface to manage systemd via the systemctl command.
 type Systemd interface {
 	DaemonReload() error
-	Enable(service string) error
-	Disable(service string) error
-	Start(service string) error
-	Stop(service string, timeout time.Duration) error
+	Enable(services ...string) error
```

Comment:
I wonder if we it would be feasible to remove the non-wait variants from the interface now? Or are they still needed in the code outside the systemd package themself?